client/src/pages/Login/DemoLogin/DemoLogin.tsx

```diff
@@ -0,0 +1,55 @@
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
+import CircularProgress from '@mui/material/CircularProgress';
+import { Formik, FormikHelpers } from 'formik';
+import useStyles from './useStyles';
+
+interface Props {
+  handleSubmit: (
+    {
+      email,
+      password,
+    }: {
+      email: string;
+      password: string;
+    },
+    {
+      setStatus,
+      setSubmitting,
+    }: FormikHelpers<{
+      email: string;
+      password: string;
+    }>,
+  ) => void;
+}
+
+export default function DemoLogin({ handleSubmit }: Props): JSX.Element {
```

Comment:
This component is very similar to `DemoLoginButton` and isn't used anywhere. Should we remove it? If not, what is this component for?

Reply:
This was my starting code for Demologinbutton but I forgot to delete this.